templates/nginx.conf.hbs

```diff
@@ -16,7 +16,7 @@ http {
 
     log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                       '$status $body_bytes_sent "$http_referer" '
-                      '"$http_user_agent" "$http_x_forwarded_for"';
+                      '"$http_user_agent" "$http_x_forwarded_for" "$http_host"';
```

Comment:
@tka85 , have you tried `$host` instead of `$http_host`?
`$http_host` is the value of the HTTP header `Host`. If ot's absent, the nothing will be logged. But `$host` is represents the name of the server requested by the client, without the port number. It only contains the hostname or IP address. It works even when request header `Host` is not set. Sounds like more flexible option.

Reply:
I didn't try `$host` but I saw that `$http_host` worked for me with rev proxying to docker containers.

Curious why you resolved it.

Reply:
I found some examples on the StackOverflow with `$http_host` after leaving this comment.